cmd/cli/uninstall_mesh.go

```diff
@@ -199,6 +182,64 @@ func (d *uninstallMeshCmd) run() error {
 			fmt.Fprintf(d.out, "OSM namespace CANNOT be deleted in a managed environment\n")
 		}
 	}
+
+	err := d.deleteClusterResources()
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
```suggestion
	err := d.deleteClusterResources()
	return err
```

Or this could even be more simplified to:

```suggestion
	return d.deleteClusterResources()
```

But I think this is likely not the most readable choice.